### PR TITLE
New parameter to add more options to the header, fixes #200

### LINKED ISSUE #200: Missing User-Agent header to get images
Some servers denied flyimg app to download the image because of missing User-Agent header.
A solution is to add this header to the InputImage::saveToTemporaryFile() function.

### PR DESCRIPTION
**Pull request by sadok-f**

### CODE REVIEW

File: config/parameters.yml [hunk 1]
```diff
@@ -36,6 +36,8 @@ aws_s3:
   region: ""
   bucket_name: ""
 
+header_extra_options: 'User-Agent: Mozilla/5.0 (Windows; U; Windows NT 6.1; rv:2.2) Gecko/20110201'
```
Comment (baamenabar) [inline config/parameters.yml]: could you add a comment or reference to the issue?

or add it in the docs

File: src/Core/Entity/OptionsBag.php [hunk 2]
```diff
@@ -146,6 +146,15 @@ public function getOption($key)
         return array_key_exists($key, $this->optionsCollection) ? $this->optionsCollection[$key] : '';
     }
 
+    /**
+     * Returns the AppParamters Object
+     * @return AppParameters
+     */
+    public function appParameters()
+    {
+        return $this->appParameters;
```
Comment (baamenabar) [inline src/Core/Entity/OptionsBag.php]: isn't it exposing too much? it does the work though

File: config/parameters.yml (hunk 1 above)
Comment (sadok-f) [reply config/parameters.yml]: Good catch! I'll add comments above it.

File: src/Core/Entity/OptionsBag.php (hunk 2 above)
Comment (sadok-f) [reply src/Core/Entity/OptionsBag.php]: No I think it is ok, and we don't need to instantiate the class again for one use case.

Comment (baamenabar) [reply src/Core/Entity/OptionsBag.php]: Agree, definitively not that

**Pull request merged**